src/model/orders/Order.java

```diff
@@ -0,0 +1,72 @@
+package model.orders;
+
+import model.Product;
+import model.persons.Client;
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.Collection;
+
+/**
+ * The Order Class has this Client and the list of OrderDetail (list of product and quantity)
+ */
+public class Order {
+    Integer id;
```

Comment:
These variables should be private

Reply:
Same here!

Shoul I change the access modifier in the field **id** in all my approved model classes?